src/main/java/pl/sda/weather/dao/ILocationDAO.java

```diff
@@ -1,9 +1,15 @@
 package pl.sda.weather.dao;
 
 
+import pl.sda.weather.model.LocationModel;
+
+import java.util.List;
 
 public interface ILocationDAO {
```

Comment:
I think better naming will be "Repository". DAO = data access object (and should be object, not interface)